src/i2c.rs

```diff
@@ -99,6 +102,7 @@ pub struct BlockingI2c<I2C, PINS> {
 }
 
 impl<PINS> I2c<I2C1, PINS> {
+    /// Creates a generic I2C1 object (on pins PB6/7 or PB8/9)
```

Comment:
```suggestion
    /// Creates a generic I2C1 object on pins PB6 and PB7 or PB8 and PB9 if remapped
```

Reply:
changed to
>  /// Creates a blocking I2C1 object on pins PB6 and PB7 or PB8 and PB9 (if remapped)
    /// embedeed-hal compatible!